VRDR/DeathRecord.cs

```diff
@@ -58,6 +58,30 @@ public class DeathRecord
         /// <summary>Condition Contributing to Death.</summary>
         private Condition ConditionContributingToDeath;
 
+        /// <summary>Create a Cause Of Death Condition </summary>
+        private Condition CauseOfDeathCondition(int index){
+                    Condition CodCondition;
+                    Console.Error.WriteLine("CauseOfDeathCondition index = " + index);
```

Comment:
Should this log be removed? Or changed to an info level log

Reply:
Doh